cli/src/commands/workspace.rs

```diff
@@ -46,7 +47,8 @@ pub(crate) enum WorkspaceCommand {
     Add(WorkspaceAddArgs),
     Forget(WorkspaceForgetArgs),
     List(WorkspaceListArgs),
-    Root(WorkspaceRootArgs),
+    #[command(hide = true)] // favor jj root
```

Comment:
Nit: I'd probably not hide the command until some way of "soft deprecation" has happened.

Reply:
In the end, we left `jj workspace root` as is, so it solves your remark.